### DISCUSSION TITLE
No rows found even for internal Budibase Users Table

### DISCUSSION THREAD

**Discussion by thecodrr** [Budibase/budibase#4031] [Help]
I am trying out Budibase and so far so good. But I can't actually do anything since adding a Data Provider always gives me "No rows found" error.

I have tried with MongoDB as well as Budibase's internal Users DB. It always shows the same error. I can notice significant network activity (when using MongoDB) but nothing shows up.

I am self-hosting Budibase and I ran the MongoDB query successfully from inside the Budibase. Maybe I am doing something wrong. Any help is appreciated.

**Comment by CovantaDataManagement** [comment]
I'm not sure if this will help you as I am new myself but I just tried creating a listing of users as a table inside a datasource on a page per the attached screenshot and it seems to work.  This is using the internal Users table.
![2022-01-14_12-25-04](https://user-images.githubusercontent.com/62119887/149558641-1a6d2d0e-dbd3-4b9a-9d27-45fe871b82ba.png)
